wire-ios/Wire-iOS/Sources/Components/Settings/SettingsPropertyFactory.swift

```diff
@@ -436,6 +436,20 @@ final class SettingsPropertyFactory {
                 }
             )
 
+        case .conversationBackground:
+            let userId = selfUser!.remoteIdentifier!
```

Comment:
todo: avoid force unwrapping

Reply:
I would prefer not to crash and log a warning than having it crash. I guess it all come to strategy. but in my opinion crashing the app should be a last resort.